### DISCUSSION TITLE
Is this supposed to work on web?

### DISCUSSION THREAD

**Discussion by ranhorev** [henninghall/react-native-date-picker#313] [Q&A] [answered]
I am getting odd errors trying to load the page via web. Is it not supposed to work on web?
Thanks

**Comment by henninghall** [comment, marked as answer]
No it isn't. Would be a nice feature though!

**Reply by nandorojo** [reply]
@henninghall what if we used the native `input` with `type="date"`? Some older browsers don't support it, like Safari, but newer ones do.

**Reply by nandorojo** [reply]
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date
